<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#150050;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton#SaveButton {
background-color:#FF4C29;
color:white;
}

QPushButton#SaveButton:hover {
    background-color:white;
color:#FF4C29;
}

QPushButton#SaveButton:pressed {
    background-color: rgb(224, 0, 0);     
}

QPushButton#AddSessionButton {
background-color:#FF4C29;
color:white;
}

QPushButton#AddSessionButton:hover {
    background-color:white;
color:#FF4C29;
}

QPushButton#AddSessionButton:pressed {
    background-color: rgb(224, 0, 0);     
}
QPushButton#BackButton {
background-color:#FF4C29;
color:white;
}

QPushButton#BackButton:hover {
    background-color:white;
color:#FF4C29;
}

QPushButton#BackButton:pressed {
    background-color: rgb(224, 0, 0);     
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FF4C29;</string>
    </property>
    <property name="text">
     <string>ENTER TRAINER DETAILS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>610</y>
      <width>808</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>SAVE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>770</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nametLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FF4C29;</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="namelineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>820</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1px solid white;
border-radius : 10px;
color : white;</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="addresstLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>67</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FF4C29;</string>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="agetLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>32</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FF4C29;</string>
    </property>
    <property name="text">
     <string>Age</string>
    </property>
   </widget>
   <widget class="QLabel" name="mobileNumbertLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>470</y>
      <width>130</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FF4C29;</string>
    </property>
    <property name="text">
     <string>Mobile Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addressLineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>820</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1px solid white;
border-radius : 10px;
color : white;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ageLineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>820</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1px solid white;
border-radius : 10px;
color : white;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="mobileNumberLineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>470</y>
      <width>820</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1px solid white;
border-radius : 10px;
color : white;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <action name="actionStudent">
   <property name="text">
    <string>Student</string>
   </property>
  </action>
  <action name="actionTrainer">
   <property name="text">
    <string>Trainer</string>
   </property>
  </action>
  <action name="actionStudent_2">
   <property name="text">
    <string>Student</string>
   </property>
  </action>
  <action name="actionTrainer_2">
   <property name="text">
    <string>Trainer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
